From dc42dfeb6848f4098adaae0a7163a768c6bceeba Mon Sep 17 00:00:00 2001
From: Stefan Roese <sr@denx.de>
Date: Thu, 6 Sep 2018 08:22:29 +0200
Subject: [PATCH] common: Add arch_misc_init() prototype to include/init.h

When implementing arch_misc_init() for the MT7620 platform I noticed
that this function prototype is missing. Lets add this prototype to the
common place (init.h) so that we use this function on all architectures.

Signed-off-by: Stefan Roese <sr@denx.de>
Cc: Daniel Schwierzeck <daniel.schwierzeck@gmail.com>
Cc: Patrick Delaunay <patrick.delaunay@st.com>
Cc: Tom Rini <trini@konsulko.com>
---
 include/init.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/init.h b/include/init.h
index f114a88963..d2a6c4d33b 100644
--- a/include/init.h
+++ b/include/init.h
@@ -157,6 +157,7 @@ void arch_setup_gd(gd_t *gd_ptr);
 /* common/board_r.c */
 void board_init_r(gd_t *id, ulong dest_addr) __attribute__ ((noreturn));
 
+int arch_misc_init(void);
 int cpu_init_r(void);
 int last_stage_init(void);
 int mac_read_from_eeprom(void);
